controllers/compliance.go

```diff
@@ -0,0 +1,108 @@
+package controllers
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/RedHatInsights/entitlements-api-go/config"
+	l "github.com/RedHatInsights/entitlements-api-go/logger"
+	"github.com/RedHatInsights/entitlements-api-go/types"
+	"github.com/getsentry/sentry-go"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+var screeningPathV1 = "/v1/screening"
+var complianceServiceName = "Export Compliance Service"
+
+var complianceFailure = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "it_export_compliance_service_failure",
+		Help: "Total number of IT export compliance service failures",
+	},
+	[]string{"code"},
+)
+var complianceTimeHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+	Name:    "it_export_compliance_service_time_taken",
+	Help:    "Export compliance service latency distributions.",
+	Buckets: prometheus.LinearBuckets(0.25, 0.25, 20),
+})
+
+func Compliance() func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, req *http.Request) {
+		start := time.Now()
+		reqBody, err := ioutil.ReadAll(req.Body)
```

Comment:
- [ ] instead of requiring a body, pull the `x-rh-identity` header from the request since that is being enforced, and build the request to compliance based off that. Then change this endpoint to a GET instead of POST